### PR TITLE
[CPU] enable conversion def_convV8 -> def_convV1

### PR DESCRIPTION
**Pull request by mandrono**
### Tickets:
 - 61223

### CODE REVIEW

**Comment by mandrono** [comment]
@dmitry-gorokhov could you take a look please?

File: ngraph/test/runtime/ie/unit_test.manifest [hunk 1]
```diff
@@ -1599,6 +1599,15 @@ IE_CPU.deformable_convolution_2D_integral_offsets_groups_and_deforgroups
 IE_CPU.deformable_convolution_2D_real_offsets_groups_basic
 IE_CPU.deformable_convolution_2D_real_offsets_groups_complex
 IE_CPU.deformable_convolution_2D_real_offsets_groups_and_deforgroups
+IE_CPU.deformable_convolution_opset8_2D_zeroed_offsets_groups_basic
```
Comment (ilyachur) [inline ngraph/test/runtime/ie/unit_test.manifest:1602]: Looks like after this PR we have some regressions. Am I right?

Comment (mandrono) [reply ngraph/test/runtime/ie/unit_test.manifest:1602]: yes, in previous implementation we fallback on reference implementation def_conv, but now we convert opset8 def_conv -> opset1 def_conv and execute version from CPU plug-in which not support ```group > 1```. As far as I now we have no real cases with ```group > 1``` so it's broke tests only. If it necessary I can add callback which will be check that ```group > 1``` and perform transformation and tests should pass.
Do we need these?

Comment (dmitry-gorokhov) [reply ngraph/test/runtime/ie/unit_test.manifest:1602]: I don't think we need it right now. Lets merge as is and enable these tests as part of PR with full Deformable_Convolution_8 support.
cc'ed @yury-intel

**Review by dmitry-gorokhov — APPROVED** [review]
LGTM. @mandrono Please rebase the PR on latest master state to get fresh CI

**Comment by mandrono** [comment]
see: https://github.com/openvinotoolkit/openvino/pull/6945

**Pull request closed without merging**